### PR TITLE
Fix: Upgrading packages with known vulns

### PR DESCRIPTION
**Pull request by MovieStoreGuy**

### CODE REVIEW

**Review by hstan — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+39.0.1
+------
+- Bump golang.org/x/net from v0.17.0 -> v0.23.0
+- Bump protobuf (protoc) from 21.5 -> 26.1
```
Comment (tiedotguy) [inline CHANGELOG.md:4]: Don't we generate and commit the protobuf stuff?  Or is it generated on compile?

**Review by tiedotguy — DISMISSED** [review]
Probably ok, but double check if we need to re-gen protobuf?

File: CHANGELOG.md (hunk 1 above)
Comment (MovieStoreGuy) [reply CHANGELOG.md:4]: I thought we did too, but looks like some of the tooling has rotted or I am missing steps.

Comment (MovieStoreGuy) [reply CHANGELOG.md:4]: Updated, just changed the comment.

**Review by irisgve — APPROVED** [review]

**Pull request merged**